ISISComputingGroup/ibex_gui
# Ticket2730 Plot multiple PVs

### Description of work

Allows plotting of multiple PVs from right clicking on a block.

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/2730

### To Test

Confirm the following:
* You can plot a block by right clicking on it
* When a plot has already been created you can add a trace to it by right clicking on a block
* The axes have the same name as the blocks
* The axes still contain the units for the PV

### Unit tests

Very UI focused, hard to unit test.

---

#### Code Review


### Final Steps
